wire-library/wire-schema/src/jvmMain/kotlin/com/squareup/wire/schema/CoreLoader.kt

```diff
@@ -44,7 +44,7 @@ actual object CoreLoader : Loader {
 
   override fun load(path: String): ProtoFile {
     if (isWireRuntimeProto(path)) {
-      FileSystem.RESOURCES.read("/".toPath() / path) {
+      CoreLoader::class.java.classLoader.asResourceFileSystem().read("/".toPath() / path) {
```

Comment:
Can you cache the result of `asResourceFileSystem()` ? It has to do a non-zero amount of work!

Reply:
I put it in a `lazy` delegate